src/main/java/seedu/address/logic/parser/DeletePersonFromModuleCommandParser.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_CODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.ParserUtil.arePrefixesPresent;
+
+import seedu.address.logic.commands.DeletePersonFromModuleCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.person.Name;
+
+/**
+ * Parses input arguments and creates a new {@code DeletePersonFromModuleCommand} object
```

Comment:
```suggestion
 * Parses input arguments and creates a new {@code DeletePersonFromModuleCommand} object.
```

Fullstop for consistency

Reply:
Thanks for pointing this out!